core/templates/pages/exploration-player-page/layout-directives/audio-bar.component.html

```diff
@@ -30,7 +30,7 @@
             <mat-progress-bar mode="indeterminate"></mat-progress-bar>
           </div>
           <div *ngIf="!audioLoadingIndicatorIsShown">
-            <oppia-audio-slider [value]="audioPlayerService.getCurrentTime()"
```

Comment:
This caused an error (screenshot attached), hence, I updated the code. 

-----

![Screenshot from 2024-07-07 00-17-05](https://github.com/oppia/oppia/assets/57531197/4f76549b-7d6f-4a4b-a835-07771b11bfc0)